### PR TITLE
Gh 1410

### PR DESCRIPTION
**Pull request by ArifPMan**
Fixed. Please review and close.

### CODE REVIEW

**Comment by numaanashraf** [comment]
@ArifPMan Please name the PR more aptly.

**Comment by ArifPMan** [comment]
Would naming the PR the same as the issue name be okay with you?

**Comment by numaanashraf** [comment]
@ArifPMan You can give a short sentence detailing what you are doing in this PR
For example: Remove outdated history behavior for GET requests

File: v6/postman/collections/managing_collections.md [hunk 1]
```diff
@@ -57,15 +57,23 @@ If you have a lot of collections, filter collections in the sidebar using the se
 
 ### Delete a collection
 
-Click the ellipsis (...)next to a collection, and select "Delete". If you didn't intend to delete the collection, you can click the **Undo** link in the notification that appears at the top of the Postman app.
+Click the ellipsis (...)next to a collection, and select "Delete". If you didn't intend to delete the collection, you can click the **Undo** link in the notification that appears at the top of the Postman app.
```
Comment (numaanashraf) [inline v6/postman/collections/managing_collections.md]: These changes seem unrelated to the issue that is being worked on #1410

**Comment by ArifPMan** [comment]
Cleaned up the file.. Please review and close...
